src/librustc/hir/mod.rs

```diff
@@ -51,6 +51,15 @@ macro_rules! hir_vec {
     );
 }
 
+macro_rules! arena_vec {
```

Comment:
This should be in `lowering.rs`.

Reply:
I did not manage to get it imported in the lowering sub-modules. I do not know this corner of the language enough.

Reply:
Should be fine, we can move it later.

Reply:
The macro definition has to be before the declaration of the modules in `lowering.rs`.